versioned_docs/version-latest/monitoring.md

```diff
@@ -0,0 +1,558 @@
+    static_configs:
+      - targets: ["localhost:9100"]
+  - job_name: 'any-other-your-job'
+    static_configs:
+      - targets: ['123.123.123.123:12345']
+
+```
+:::note
+Remember to allow connections between your cloud server and monitored servers at UFW on the appropriate ports. 
+:::
+---------------------------------------------------------------------------------------------------------------------------------------------
+
+## 3. Node Exporter.
+
+### Creat a User:
+```
+sudo useradd -M -r -g prometheus -s /sbin/nologin node_exporter
+```
+### installing
+
+From [Prometheus homepage](https://prometheus.io/download) download latest version of binary
+Extract:
+```bash
+tar xvfz node_exporter-*-amd64.tar.gz
+```
+
+Copy binary:
+```bash
+cd node_exporter-*-amd64
+sudo cp node_exporter-*md64/node_exporter /usr/local/bin
+sudo chown node_exporter:node_exporter /usr/local/bin/node_exporter
+```
+
+
+### NodeExporter Service
+
+```bash
+sudo nano /etc/systemd/system/node_exporter.service
+```
+Copy-paste this into the file:
+```
+[Unit]
+Description=Node Exporter
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=node_exporter
+Group=node_exporter
+Type=simple
+ExecStart=/usr/local/bin/node_exporter
+
+[Install]
+WantedBy=multi-user.target
+```
+
+### Service Management
+```
+sudo systemctl daemon-reload
+sudo systemctl enable node_exporter
+
+sudo systemctl start node_exporter
+sudo systemctl status node_exporter
+```
+### Add NodeExporter to prometheus.yml
+
+```bash
+sudo nano /etc/prometheus/prometheus.yml
+```
+To the end of the file add this section:
+```
+  - job_name: 'node_exporter'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9100']
+```
+We already did it , and the job named "FarmerServer"
+
+------------------------------------------------------
+
+## 4. Scraping Metrics from Subspace Node
+
+The Subspace Node designed so that it provides it's metrics on port 9615.
+The following yaml file is designed to scrape Subspace Node metrics when the Node is ran in Docker.
+
+To use this: 
+
+1) complete upper part of docker-compose.yml  and add:
+```
+    "--prometheus-external"
+    "--prometheus-port", "9615"
+```
+and
+```
+            expose:
+              - "9615:9615"
+```
+2) complete lower part (grafana, prometheus and node exporter)
+
+### Docker-Compose.yml Template for Monitoring Subspace Node  
+ (the template is dated Sept. 25)
+		
+    > 
+    version: "3.7"
+
+    services:
+
+    # ********************************************************************************
+    # The following section should be copied from the official Subspace Docker Compose
+    # guide at https://github.com/subspace/subspace/blob/main/docs/farming.md
+    #*********************************************************************************
+
+    node:
+        container_name: subspace-node
+        # Replace `snapshot-DATE` with latest release (like `snapshot-2022-apr-29`)
```

Comment:
Let's update the whole command for running a node to point to a newer release and gemini 3g to not confuse anyone. Thanks!